hivemind_core/__main__.py

```
@@ -45,8 +110,47 @@ def main():
+        error_msg = f"Failed to start HiveMind service: {e}"
+        LOG.exception(error_msg)
+
+        # Start admin UI for error diagnostics if requested
+        if args.with_admin:
+            time.sleep(0.5)
+            start_admin_with_error(
+                host=args.admin_host,
+                port=args.admin_port,
+                error=e
+            )
+            LOG.warning(f"Admin UI started for error diagnostics")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)